release_notes_generator/action_inputs.py

```diff
@@ -38,49 +42,114 @@
 
 
 class ActionInputs:
+    """
+    A class representing the inputs provided to the GH action.
+    The class is responsible for handling the inputs provided to the GH action.
+    """
+
     @staticmethod
     def get_github_repository() -> str:
+        """
+        Get the GitHub repository from the action inputs.
+
+        @return: The GitHub repository.
```

Comment:
Last time we were talking about documentation, we decided, that we won't write a `@return` for getters.

Reply:
Yeah, my bad. I forgot about it in this big wave of changes.